src/components/MovieSearch.js

```diff
@@ -42,12 +42,12 @@ function MovieSearch() {
             },
           })
           .then((res) => {
-            // console.log(res.data);
+            // console.log(res);
             setResources(res.data);
-          })
-          .catch((err) => {
-            setError(err);
-            //console.log(err)
+            if(res.data.Error) {
```

Comment:
use optional chaining like this:

if(res?.data?.Error) {

Reply:
done